specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -705,14 +705,17 @@
       "type": "object",
       "properties": {
         "from": {
+          "description": "The start index to fetch sessions.",
```

Comment:
>sessions [](start = 51, length = 8)

spark batch Job?

Reply:
I add description following [livy documatation](https://livy.apache.org/docs/latest/rest-api.html)(search GET /batches in page)

Is it ok for different from this doc?

Reply:
I think that the minor change is acceptable and necessary to fit in our sdk. Although the class name is SparkBatchJobCollection.

---
In reply to: [455539008](https://github.com/Azure/azure-rest-api-specs/pull/10121#discussion_r455539008) [](ancestors = 455539008)